modules/auxiliary/gather/zoomeye_search.rb

```diff
@@ -3,21 +3,28 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
-
-
 class MetasploitModule < Msf::Auxiliary
   include Msf::Auxiliary::Report
 
-
   def initialize(info={})
     super(update_info(info,
       'Name'        => 'ZoomEye Search',
       'Description' => %q{
         The module use the ZoomEye API to search ZoomEye. ZoomEye is a search
         engine for cyberspace that lets the user find specific network
         components(ip, services, etc.).
+        Mind to enclose the whole request with quotes and limit the span of filters:
+        `set zoomeye_dork 'country:"france"+some+query'`
+
+        Setting facets will output a simple report on the overall search. It's values are:
+        Host search: app, device, service, os, port, country, city
+        Web search: webapp, component, framework, frontend, server, waf, os, country, city
+
+        Possible filters values are:
+        Host search: app, ver, device, os, service, ip, cidr, hostname, port, city, country, asn
+        Web search: app, header, keywords, desc, title, ip, site, city, country
       },
-      'Author'      => [ 'Nixawk' ],
+      'Author'      => [ 'Nixawk, Yvain' ],
```

Comment:
This is still not quite right, it should be an array with two separate strings like this: `[ 'Nixawk', 'Yvain' ],`.